src/bp/ui-studio/src/web/views/OneFlow/diagram/index.tsx

```diff
@@ -201,6 +205,12 @@ class Diagram extends Component<Props> {
     this.searchRef = React.createRef()
   }
 
+  isNodeSelected = (nodeId: string) => {
+    const queryParams = new URLSearchParams(window.location.search)
+
+    return queryParams.get('highlightedNode') === nodeId
```

Comment:
There is already a complex logic to highlight nodies in `manager.ts` (shouldHighlightNode), can you move this logic there instead ? It would remove the need to add your "isSelected" method because it's already handled there.

Also this seems to break the individual node highlighting 
![image](https://user-images.githubusercontent.com/42552874/93241615-9ad1ac00-f753-11ea-8228-4d74b3471be1.png)